pkgs/by-name/zo/zoom-us/package.nix

```diff
@@ -83,9 +82,9 @@ let
 
   unpacked = stdenv.mkDerivation {
     pname = "zoom";
-    version = versions.${system} or throwSystem;
+    version = versions.${system} or "";
 
-    src = srcs.${system} or throwSystem;
+    src = srcs.${system};
```

Comment:
Can we keep `throwSystem` here?  If someone uses `NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1` to force building for another system, the error message "unsupported system" is still more useful than "attribute '${SYSTEM}' missing".

...unless this hamstrings the purpose of this pull request in some way.